tasks/README.md

```diff
@@ -24,6 +24,8 @@
     <task_id>,
     <task_id>
   ],
+  "level": 1 | 2 | 3 | 4 | 5,
+  "category": "FRONTEND | BACKEND | SYSTEM DESIGN",
```

Comment:
I think the category will not be just these three, there will be other categories as well

Reply:
okay should I give any generic type as string in UPPERCASE, <architecture>